games-fps/yamagi-quake2/yamagi-quake2-8.00.ebuild

```diff
@@ -0,0 +1,151 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit desktop wrapper
+
+CTF_V="1.08"
+ROGUE_V="2.08"
+XATRIX_V="2.09"
+REF_VK_V="1.0.1"
+
+DESCRIPTION="Quake 2 engine focused on single player"
+HOMEPAGE="https://www.yamagi.org/quake2/"
+SRC_URI="https://deponie.yamagi.org/quake2/quake2-${PV}.tar.xz
+	ctf? ( https://deponie.yamagi.org/quake2/quake2-ctf-${CTF_V}.tar.xz )
+	rogue? ( https://deponie.yamagi.org/quake2/quake2-rogue-${ROGUE_V}.tar.xz )
+	xatrix? ( https://deponie.yamagi.org/quake2/quake2-xatrix-${XATRIX_V}.tar.xz )
+	vulkan? ( https://github.com/yquake2/ref_vk/archive/refs/tags/v${REF_VK_V}.tar.gz ->
+		quake2-ref_vk-${REF_VK_V}.tar.gz )
+"
+S="${WORKDIR}/quake2-${PV}"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~x86"
+IUSE="+client ctf dedicated openal +opengl rogue softrender vulkan xatrix"
+REQUIRED_USE="
+	|| ( client dedicated )
+	client? ( || ( opengl softrender vulkan ) )
+"
+
+COMMON_DEPEND="
+	client? (
+		media-libs/libsdl2[video]
+		!openal? ( media-libs/libsdl2[sound] )
+		opengl? (
+			media-libs/libsdl2[opengl]
+			virtual/opengl
+		)
+	)
+"
+DEPEND="${COMMON_DEPEND}
+	client? ( vulkan? ( dev-util/vulkan-headers ) )
+"
+RDEPEND="${COMMON_DEPEND}
+	client? ( openal? ( media-libs/openal ) )
```

Comment:
I'm pretty sure openal needs to be in COMMON_DEPEND even if loaded at runtime, I see this package use `#include <AL/al.h>` openal headers.

Reply:
Oh, and I just realized curl is not listed anywhere.. it uses curl headers too.

What's up with handling curl? It's not an implicit system dependency, it can very well be missing on a minimal system.